env-builder/src/Include.mk

```
@@ -0,0 +1,26 @@
+# ============================================================================ #
+#
+#
+#
+# ---------------------------------------------------------------------------- #
+
+SRC_DIR := $(call getMakefileDir)
+
+
+# ---------------------------------------------------------------------------- #
+
+_BUILT_SRCS := buildenv.nix get-env.sh
+_BUILT      := $(patsubst %,$(SRC_DIR)/%.gen.hh,$(_BUILT_SRCS))
+_BUILT_SRCS =
+
+BUILT_SRCS       += $(_BUILT)
+env-builder_SRCS += $(_BUILT)
```

```suggestion
env_builder_SRCS += $(_BUILT)
```

`make` won't let you use `-` in a variable name as far as I know.

---

Be sure to change the other cases where the variable is used as well

---

didnt complain for me, but did the change